### ISSUE TITLE
Process breaks with error: get_series_to_exclude KeyError: 'title'

### ISSUE THREAD

**Issue by alejakun** [haijeploeg/excludarr#49]
**Describe the bug**
Process breaks at 51% with the following stack trace:

**To Reproduce**
I'm running on docker for amd64 (latest) with the following command:

`docker run -it --rm --env-file excludarr.env haijeploeg/excludarr:latest sonarr exclude -a not-monitored -d -e  --progress`

My env. file looks like this:

```
GENERAL_FAST_SEARCH=true
GENERAL_LOCALE=es_MX
GENERAL_PROVIDERS="Netflix, Amazon Prime Video, Apple TV Plus"

TMDB_API_KEY=<EDITED>

RADARR_URL=https://radarr.myediteddomain.com
RADARR_API_KEY=<EDITED>
RADARR_VERIFY_SSL=true
# RADARR_EXCLUDE=","

SONARR_URL=https://sonarr.myediteddomain.com
SONARR_API_KEY=<EDITED>
SONARR_VERIFY_SSL=true
SONARR_EXCLUDE=""
```

**Expected behavior**

For the application to finish and request to confirm show deletion.

**Versions**
Docker: haijeploeg/excludarr:latest (amd64)

**Debug logging**

```
Traceback (most recent call last):
  File "/usr/local/bin/excludarr", line 33, in <module>
    sys.exit(load_entry_point('excludarr==1.0.2', 'console_scripts', 'excludarr')())
  File "/usr/local/lib/python3.10/site-packages/excludarr-1.0.2-py3.10.egg/excludarr/main.py", line 68, in cli
  File "/usr/local/lib/python3.10/site-packages/typer/main.py", line 214, in __call__
    return get_command(self)(*args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/typer/main.py", line 500, in wrapper
    return callback(**use_params)  # type: ignore
  File "/usr/local/lib/python3.10/site-packages/excludarr-1.0.2-py3.10.egg/excludarr/commands/sonarr.py", line 69, in exclude
  File "/usr/local/lib/python3.10/site-packages/excludarr-1.0.2-py3.10.egg/excludarr/core/sonarr_actions.py", line 194, in get_series_to_exclude
KeyError: 'title'
```

**Comment by haijeploeg** [comment]
Hi, I see that you are using version 1.0.2 of excludarr, this issue should be fixed in version v1.0.5. You should re download your docker container using the `latest` tag or image version `v1.0.5`.

**Comment by alejakun** [comment]
Thanks, apparently I had an old version of the image and just had to `docker pull` to get the latest version. Thanks!
